### ISSUE TITLE
Finishing a map doesn't increase the map playcount

### ISSUE THREAD

**Issue by Apis035** [ppy/osu-stable-issues#1016]
If you played until the end of [this difficulty](https://osu.ppy.sh/beatmapsets/1399444#osu/2887160), the playcount on the beatmap page will not increase and displays a notification "An error occured during score calculation". However if you exited before the map ends, it does increasing playcount (As long as you have hit the first circle).

**Comment by peppy** [comment]
Could you please attach log files from your osu! folder after playing through and causing the error?

**Comment by Apis035** [comment]
[osu!auth.log](https://github.com/ppy/osu-stable-issues/files/9243043/osu.auth.log)
[runtime.log](https://github.com/ppy/osu-stable-issues/files/9243044/runtime.log)
[session.log](https://github.com/ppy/osu-stable-issues/files/9243045/session.log)
[update_success.log](https://github.com/ppy/osu-stable-issues/files/9243046/update_success.log)

Sorry, forgot to mention.
The map has 2B sliders and forced breaks created by editing the .osu file. I guess that what makes osu fail to calculate the score. But even if the score failed to calculate, the playcount should be still updated like it did when we played the map and quit before the map ends.

**Comment by peppy** [comment]
Yeah that's likely the issue. I'm not sure this is worth investigating further as this is a graveyarded beatmap.

Your log files do look like they are from a fresh session though, and don't contain anything of use.
